deploy/crds/awx.ansible.com_v1beta1_backup_cr.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: awx.ansible.com/v1beta1
+kind: Backup
+metadata:
+  name: example-awx
+  namespace: example-awx
+spec:
+  tower_backup_pvc: ''
+  tower_backup_size: ''
+  tower_postgres_storage_class: ''
```

Comment:
`_postgres_` or `_backup_` ?

Reply:
Our installer role also has this variable, so I thought it might be better to call it `tower_postgres_storage_class` for consistency across the operator.  But I could also see the argument for consistency in the role.  

It is probably better to have these vars be distinct (and consistently named within the role), I'll change this to `tower_backup_storage_class`